src/test/resources/alma/HT005207972.json

```diff
@@ -107,7 +107,7 @@
     },
     "resultOf" : {
       "type" : [ "CreateAction" ],
-      "endTime" : "2021-01-05T14:18:03",
```

Comment:
While these data indeed changes, I recommend to not add it: the java unit tests ignores the "endTime", so test are successful - and if we don't have these resources in our commits the diff is smaller. I know this may be confusing, but the tests uses the productive morphs where this field is created.
Maybe it would be better to filter this field out before writing these test files so that they wouldn't appear here. Feel free to open an issue if this would be made things much easier.

Reply:
We have two options:
1. remove these modifications in the commit and force push them
2. leave it as it is and just remember to not add them in the future

Reply:
How could I prevent this in the future since the endTime is always changing when testing?

Reply:
You can't. Just don't add them if you see that only the endTime was changed.

Reply:
ah, okay, I see what you mean. I should only add those testfiles that change concerning the issue that I am working at. Then lets keep it like this, for this time. Next time I will only `git add` the relevant files.

Reply:
+1